### PR TITLE
Replace hardcoded values

### PR DESCRIPTION
**Pull request by Xenwar**
Replace hardcoded values

Signed-off-by: Anwar Hassen <anwar.hassen@est.tech>

### CODE REVIEW

File: capo/configure.sh [hunk 1]
```diff
@@ -57,7 +57,6 @@ clouds:
   "${OS_CLOUD_NAME}":
     auth:
       auth_url: "${OS_AUTH_URL}"
-      project_name: Default Project 37137
```
Comment (lentzi90) [inline capo/configure.sh:60]: Do we not need this? Would it make sense to parametrize it instead?
```
project_name: "${PROJECT_NAME}"

Comment (Xenwar) [reply capo/configure.sh:60]: We do and we still have it at line 66. removed the duplicate one

Comment (Xenwar) [reply capo/configure.sh:60]: And, the variable names are in the format `OS_`

Comment (lentzi90) [reply capo/configure.sh:60]: Ah thanks for pointing it out! :slightly_smiling_face:

**Review by lentzi90 — APPROVED** [review]

**Pull request merged**